core/src/main/java/io/aiven/klaw/model/response/EnvUpdatedStatus.java

```diff
@@ -1,11 +1,12 @@
 package io.aiven.klaw.model.response;
 
+import io.aiven.klaw.model.enums.ClusterStatus;
 import jakarta.validation.constraints.NotNull;
 import lombok.Data;
 
 @Data
 public class EnvUpdatedStatus {
   @NotNull private String result;
 
-  @NotNull private String envstatus;
+  @NotNull private ClusterStatus envStatus;
 }
```

Comment:
The old angular UI does not liking this, it returns an undefined.

It looks like in envs.js it is looking for output.envstatus but it is returned as envStatus
just a minor update required
 ![image](https://github.com/aiven/klaw/assets/121855584/4d7474df-cc91-4505-9b2f-91c638752069)

Reply:
Did you check that with the most current state? I saw this this morning and fixed it (at least locally it worked)
https://github.com/aiven/klaw/pull/1469/commits/7d23d0957de3cbcc68024d73b74403eed1e46c9a

Reply:
I see that the update is there now, i must not have pulled the latest update properly sorry I'll give it a quick test and approve :)